core-java-modules/core-java-collections-list-5/src/test/java/com/baeldung/java/list/ListContainsElementFromOtherListTest.java

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.java.list;
+
+import static java.util.stream.Collectors.toSet;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.junit.Test;
+
+import com.baeldung.list.Country;
+
+public class ListContainsElementFromOtherListTest {
+
+    final private List<String> listOfLetters = Arrays.asList("a", "b", "c", "d");
+    final private List<String> listOfLettersWithOverlap = Arrays.asList("d", "e", "f", "g");
+    final private List<String> listOfCities = Arrays.asList("London", "Berlin", "Paris", "Brussels");
+
+    @Test
+    public void givenWholeValuesToCompare_whenUsingCollectionsDisjoint_thenDetectElementsInTwoLists() {
+        boolean shouldBeTrue = !Collections.disjoint(listOfLetters, listOfLettersWithOverlap);
+        assertTrue(shouldBeTrue);
+
+        boolean shouldBeFalse = !Collections.disjoint(listOfLetters, listOfCities);
+        assertFalse(shouldBeFalse);
+    }
+
+    @Test
+    public void givenWholeValuesToCompare_whenUsingStreams_thenDetectElementsInTwoLists() {
+        boolean shouldBeTrue = listOfLetters.stream()
+          .anyMatch(listOfLettersWithOverlap::contains);
+        assertTrue(shouldBeTrue);
+
+        boolean shouldBeFalse = listOfLetters.stream()
+          .anyMatch(listOfCities::contains);
+        assertFalse(shouldBeFalse);
+    }
+
+    @Test
+    public void givenWholeValuesToCompare_whenUsingApacheCollectionUtils_thenDetectElementsInTwoLists() {
+        boolean shouldBeTrue = CollectionUtils.containsAny(listOfLetters, listOfLettersWithOverlap);
+        assertTrue(shouldBeTrue);
+
+        boolean shouldBeFalse = CollectionUtils.containsAny(listOfLetters, listOfCities);
+        assertFalse(shouldBeFalse);
+    }
+
+    @Test
+    public void givenWholePorpertiesInObjectsToCompare_whenUsingStreams_thenDetectElementsInTwoLists() {
```

Comment:
let's remove the **Whole** word
also please fix the typo with **Porperties** -> **Properties**

Reply:
I've taken **Whole** out of all of the test names and fixed the typo